### PR TITLE
Try bumping GDAL version

### PR DESCRIPTION
**Pull request by SrNetoChan**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
We need to rebuild some of the dependencies to fix the conflicts there.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by SrNetoChan** [comment]
Can any of you explain why bumping the build number resulted in conflits, while before it didn't have any failing tests?

**Comment by ocefpaf** [comment]
> Can any of you explain why bumping the build number resulted in conflits, while before it didn't have any failing tests?

It was the re-render that moved the build from gdal 2 to gdal 3. If I got it right just the proj rename now should fix it.

**Comment by SrNetoChan** [comment]
No, I was the one starting that. I was trying to bump both gdal and proj. I have changed the gdal to 3 and it build correctly. But then I needed to bump the build version and it failed. Hence my question.

**Comment by ocefpaf** [comment]
> No, I was the one starting that. I was trying to bump both gdal and proj. I have changed the gdal to 3 and it build correctly. But then I needed to bump the build version and it failed. Hence my question.

I'm having trouble navigating the Azure logs on my phone but the gdal bump without the re-render did not change anything. I see that your bump, before the re-render, also failed but I cannot see the problem at the moment. I'll investigate the logs tomorrow morning.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,8 +1,3 @@
-gdal:
-  - 2.4.2
-libgdal:
-  - 2.4.2
 qt:
   - '5.9'
```
Comment (ocefpaf) [inline recipe/conda_build_config.yaml]: @SrNetoChan do we still need qt 5.9 or can we try 5.12?

File: .ci_support/linux_python3.6.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ docker_image:
 expat:
 - '2.2'
 gdal:
-- 2.4.2
+- '3.0'
```
Comment (ocefpaf) [inline .ci_support/linux_python3.6.yaml]: This is the global pinning, safer than the right one that we had before.

**Comment by ocefpaf** [comment]
> No, I was the one starting that. I was trying to bump both gdal and proj. I have changed the gdal to 3 and it build correctly. But then I needed to bump the build version and it failed. Hence my question.

Now that I'm on a desktop I can see that the failure in the bump commit was only the fact that the CIs got cancelled, probably due to the next commit. See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=92475&view=results

**Pull request merged**